ajax/libs/ammaps/package.json

```diff
@@ -1,14 +1,39 @@
 {
   "name": "ammaps",
   "filename": "ammap.js",
-  "version": "3.13.0",
```

Comment:
Version should not be removed here, as this is the current version that we have on the CDN.